agent-nodejs-12/Dockerfile.localdev

```diff
@@ -0,0 +1,32 @@
+FROM quay.io/openshift/origin-jenkins-agent-base:v4.0
+
+MAINTAINER Akram Ben Aissi <abenaiss@redhat.com>
```

Comment:
I would like to change all these by:
MAINTAINER OpenShift Developer Services <openshift-dev-services+jenkins@redhat.com>

can we start with with this one?